src/libNode/DSBlockProcessing.cpp

```diff
@@ -239,29 +239,30 @@ bool Node::LoadShardingStructure(
 
     const map<PubKey, Peer>& my_shard = shards.at(m_myShardID);
 
-    m_myShardMembers.clear();
+    // m_myShardMembers->clear();
+    m_myShardMembers.reset(new std::deque<pair<PubKey, Peer>>);
```

Comment:
Because this function will be used when a DS is down graded to a shard node, which requires the m_myShardMembers to be initialized with a new instance.

Reply:
Got it, please note next PR (downgrade) here for tracking.